src/main/java/org/jabref/gui/preferences/websearch/WebSearchTab.fxml

```diff
@@ -25,6 +27,25 @@
         <CheckBox fx:id="useCustomDOI" text="%Use custom DOI base URI for article access"/>
         <TextField fx:id="useCustomDOIName" HBox.hgrow="ALWAYS"/>
     </HBox>
+    <Label styleClass="sectionHeader" text="%Catalogues used for Search Selected"/>
```

Comment:
Can we have quotes here? (I hope, our [toolchain](https://devdocs.jabref.org/code-howtos/localization.html) can manage)

```suggestion
    <Label styleClass="sectionHeader" text="%Catalogues used for &quot;Search Selected&quot;"/>
```

Reply:
Use single quotes